modules/cp4ba/files/operator_shared_pvc.yaml

```diff
@@ -1,4 +1,4 @@
-###############################################################################
+##############################################################################
```

Comment:
This file should not be here.  It is a template that needs the value for `cp4ba_project_name` passed in.